docs/security/authentication/active-directory/index.md

```diff
@@ -4,37 +4,40 @@ description: Octopus Deploy can use Windows credentials to identify users.
 position: 0
 ---
 
-Octopus Deploy can use Windows credentials to identify users. This option is chosen during installation of the Octopus Server, or can be configured later.
+Octopus Deploy can authenticate users using Windows credentials. Windows AD authentication can be chosen during installation of the Octopus Server, or later through the configuration.
 
 :::hint
 **Domain user required during setup**
-When you go through the Octopus setup wizard, or run the commands below to switch to AD authentication mode, make sure you are signed in to Windows as a domain user. If you are signed in as a local user account on the machine (a non-domain user) you won't be able to query Active Directory, so setup will fail.
+When setting AD Authentication, either via the Octopus setup wizard or running the commands outlined below to switch to AD authentication mode, make sure you are signed in to Windows as a domain user. If you are signed in as a local user account on the machine (a non-domain user) you won't be able to query Active Directory, so setup will fail.
 :::
 
-## Active Directory sign in options {#ActiveDirectoryauthentication-ActiveDirectorysigninoptions}
-
+## Active Directory Sign-In options {#ActiveDirectoryauthentication-ActiveDirectorysigninoptions}
 If you are using Active Directory Authentication with Octopus, there are two ways to sign in.
 
 1. Integrated authentication
 2. Forms-based
 
-## Integrated authentication {#ActiveDirectoryauthentication-Integratedauthentication}
+## Authentication Schemes
+By default, Active Directory Authentication will use NTLM as the Authentication Scheme. In many circumstances, you can also configure Octopus to use Kerberos for authentication. 
 
-The easiest way to sign in when using Active Directory is to click the *Sign in with a domain account* link.
+## Integrated authentication {#ActiveDirectoryauthentication-Integratedauthentication}
+The easiest way to sign in when using Active Directory is to use Integrated Authentication. 
+This allows a one-click option to *Sign in with a domain account* as pictured below.
 
 ![Login Screen](images/ad-integrated.png "width=500")
 
-This will instruct the Octopus Server to issue a browser challenge. If you are signed in on Windows, your Windows credentials will be automatically used to sign you in.
+This will instruct the Octopus Server to issue a browser challenge. NTLM Authentication usually doesn't require much configuration except for allowing NTLM to be used in your network. This is usually on by default.
```

Comment:
I like that we're giving some thought to default behaviour as that lays out the 'happy path' for the user.

One thing that stood out here is that using words like `usually` leave the reader wondering what the edge cases are. Are there situations where I will need to provide a lot of configuration to make NTLM work? I suspect that there isn't so there is possibly unneeded ambiguity here. 

Please consider removing `usually` if NTLM is the default behaviour, or providing a little more detail around those instances where it isn't.

```suggestion
This will instruct the Octopus Server to issue a browser challenge. NTLM Authentication doesn't require much configuration except for allowing NTLM to be used in your network. This is on by default.
```